sbt-plugin-test/build.sbt

```diff
@@ -12,8 +12,12 @@ val baseSettings = versionSettings ++ Seq(
     "org.scala-js" %% "scalajs-jasmine-test-framework" % scalaJSVersion % "test"
 )
 
+lazy val crossProjectRefJS = ProjectRef(file("crossProjectRef"), "crossProjectRefJS")
+
+lazy val crossProjectRefJVM = ProjectRef(file("crossProjectRef"), "crossProjectRefJVM")
```

Comment:
Please add a comment explaining what these two things test, also reference the issue number.

Further, consider grouping them together, without an empty line between (what we usually do for JS/JVM pairs).